### ISSUE TITLE
Doesn't work on arm (armhf-ec100)

### ISSUE THREAD

**Issue by mwleeds** [flathub/org.learningequality.Kolibri#12]
On an Endless Mini running Endless OS 3.8.4, I can install the Kolibri flatpak and launch it but then after the bird logo goes away I am left with a blank white screen indefinitely.
```
$ flatpak info org.learningequality.Kolibri

Kolibri - Offline education technology platform

          ID: org.learningequality.Kolibri
         Ref: app/org.learningequality.Kolibri/arm/stable
        Arch: arm
      Branch: stable
     Version: 1.0
     License: MIT
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 359.0 MB
     Runtime: org.gnome.Platform/arm/3.36
         Sdk: org.gnome.Sdk/arm/3.36

      Commit: 550360361703fcd20ec4343088e32cc2688d61b81c0b255047642980a98fa30e
      Parent: f6d4c6e1ffc5a37fa533549d0ab46cf3057b59042eceae2633ac24c0b634af6b
     Subject: Update kolibri-installer-gnome (21b1a3ef)
        Date: 2020-05-30 05:04:24 +0000
```
`flatpak run org.learningequality.Kolibri` output:
[kolibri.log](https://github.com/flathub/org.learningequality.Kolibri/files/4905536/kolibri.log)

**Comment by wjt** [comment]
On Endless's internal bug tracker this was diagnosed as an issue in the GNOME runtime, and @dylanmccall submitted https://gitlab.gnome.org/GNOME/gnome-build-meta/-/merge_requests/727 to fix it.

**Comment by wjt** [comment]
Believed to be resolved with the GNOME 3.36.5 runtime release:

```
        ID: org.gnome.Platform
       Ref: runtime/org.gnome.Platform/arm/3.36
      Arch: arm
    Branch: 3.36
Collection: org.flathub.Stable
  Download: 291,7 MB
 Installed: 739,1 MB

    Commit: 995c980919412a60216b371e23791f5073cfe7456249632d8ad399b824fedc5c
    Parent: 16f0e996e01d9fb5f20d889439b815c4d5a73ba37211990826d2c6051c99ce6f
   Subject: Export org.gnome.Platform
      Date: 2020-08-11 18:27:11 +0000
```

**Comment by mwleeds** [comment]
Yes, this is fixed now.
